### DISCUSSION TITLE
Clicking on elements and entering information into forms

### DISCUSSION THREAD

**Discussion by JonWakefield** [seleniumbase/SeleniumBase#2275] [General]
Hello,
I apologize if my question is elementary but I have been looking on the SeleniumBase Github and following this Youtube Video (https://www.youtube.com/watch?v=5dMFI3e85ig&list=FLki3wsqfwOuzq_63rBNFjBQ&index=3&t=735s&ab_channel=MichaelMintz)
But I have been unable to find any information to help me.

What I am trying to do: 
I would like to navigate a website being undetected (enter information, click on a button, etc. simple actions). 
I am curious which methods I should use when I want to click or enter information onto a webpage to remain undetected?
Previously, for clicking on elements, I was using:
`driver.find_element().click()`
Is this still a valid way to perform actions on a website while remaining undetected?

My code starts like this:

`driver = Driver(uc=True) `
`driver.get(url)`
`// Perform undetected actions here ...`

( If I should initialize the driver a different way, please let me know)

I saw the method `driver.uc_click()` get mentioned on these forms but I have not been able to find anymore supporting documentation on this method.

To Recap: Just looking for best ways to remain undetected on a website while clicking and entering information. Also any supporting docs would be appreciated!

Also: This library has been incredibly helpful and a big improvement from undetected-chrome so thanks for all of y'all's hard work! It is very much appreciated.

**Comment by mdmintz** [comment]
Hello! Here are some examples of basic UC Mode detection-avoidance when loading a URL:

* https://github.com/seleniumbase/SeleniumBase/blob/master/examples/verify_undetected.py
* https://github.com/seleniumbase/SeleniumBase/blob/master/examples/uc_cdp_events.py
* https://github.com/seleniumbase/SeleniumBase/blob/master/examples/raw_uc_mode.py

UC Mode also has these newer examples where clicking is required:
* https://github.com/seleniumbase/SeleniumBase/blob/master/examples/raw_turnstile.py
* https://github.com/seleniumbase/SeleniumBase/blob/master/examples/raw_form_turnstile.py

**Reply by JonWakefield** [reply]
Thanks That Helps!
